tests/test_consdbClient.py

```diff
@@ -0,0 +1,140 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import responses
+
+from lsst.summit.utils import ConsDbClient
+
+
+@pytest.fixture
+def client():
+    return ConsDbClient("http://example.com/consdb")
+
+
+def test_table_name(client):
+    instrument = "latiss"
```

Comment:
I know this isn't the real DB, but just noting here that I'm hoping that the instrument names in the real thing match the ones in the butler on case, _i.e._ they're like `LATISS`, `LSSTCam`, `LSSTComCam` etc. Or are they all lower-cased? I guess that wouldn't be the end of the world, though having them be the same would probably be preferable.

Reply:
Since they're used in the database without quoting, they can actually be any case.